sdk/src/transaction.rs

```diff
@@ -293,7 +293,8 @@ mod tests {
             14, 229, 239, 119, 93, 5, 218, 161, 35, 3, 33, 0, 36, 100, 158, 252, 33, 161, 97, 185,
             62, 89, 99, 195, 250, 249, 187, 189, 171, 118, 241, 90, 248, 14, 68, 219, 231, 62, 157,
             5, 142, 27, 210, 117,
-        ])).expect("fu");
+        ]))
+        .expect("fu");
```

Comment:
oops, that looks like something I would have added during testing and forgotten to remove.  Can you please strike this :)

Reply:
haha, I just ran "cargo fmt". But sure thing - I'll upload that patch after the 1.31.0 docker image finishes uploading. I accidentally uploaded another 1.30.1 (README tells me to run ./build.sh - I assumed it did magical things).